config/config.go

```diff
@@ -334,12 +334,12 @@ type Node struct {
 		Explorer bool
 		// UpdateMetricsInterval is the interval between updates of the
 		// API metrics
-		UpdateMetricsInterval Duration
+		UpdateMetricsInterval Duration `validate:"required_with=Address"`
```

Comment:
I don't understand the `"required_with=Address"` requirement.  This field is mandatory no matter if `Address` is set or not.  Same for `UpdateRecommendedFeeInterval`.  What does `"required_with=Address"` do?

Reply:
Ok I've now seen that you added this to follow the previous implementation.   At some point we decoupled the API from the node, and with that change we made these two parameters (`UpdateMetricsInterval` and `UpdateRecommendedFeeInterval`) compulsory, but we forgot to update the code that checks these parameters.  So I think the proper solution would be to just set them both with `validate:"required"`